db/migrate/20201204004647_create_device_story_comments.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateDeviceStoryComments < ActiveRecord::Migration[5.2]
+  def change
+    create_table :device_story_comments do |t|
+      t.references :device_story, foreign_key: true
+      t.references :user, foreign_key: true
+      t.text :text
```

Comment:
You might want to have a validation to check size fo `text` in `DeviceStoryComment` because PostgresSQL accepts up to 1 GB (https://wiki.postgresql.org/wiki/FAQ#What_is_the_maximum_size_for_a_row.2C_a_table.2C_and_a_database.3F). I don't think that we would want comments in such size.